configs/ci/roles/ros.json

```diff
@@ -0,0 +1,35 @@
+{
+    "roles": [
+      {
+        "name": "Resource Optimization Administrator",
+        "display_name": "Resource Optimization administrator",
```

Comment:
We don't need `display_name` here. This is just a mechanism for overriding the default name if necessary, otherwise RBAC will automatically set `display_name` to `name`.